sys/mips/mips/bus_space_generic.c

```diff
@@ -311,30 +311,30 @@ generic_bs_rm_1(void *t, bus_space_handle_t bsh,
 
 void
 generic_bs_rm_2(void *t, bus_space_handle_t bsh,
-    bus_size_t offset, uint16_t *addr, size_t count)
+    bus_size_t offset, uint16_t *addr, bus_size_t count)
 {
-	bus_addr_t baddr = bsh + offset;
+	uintptr_t baddr = bsh + offset;
```

Comment:
These diffs all break CNMIPS.

Reply:
I mostly don't care about CNMIPS (I think it's abandonware upstream), but it does seem like this should maybe stay as a bus_addr_t for CNMIPS.  You can maybe use a typedef up in the #ifdef's for CNMIPS earlier that define the `rd16` helpers to avoid using an #ifdef here in each function.